README.md

```diff
@@ -83,15 +83,13 @@ be:
 
 - Relative to the import path:
 
-	protoc --go_out=. inputs/x.proto
-	\# writes ./github.com/golang/protobuf/p/x.pb.go
+	protoc --go_out=. inputs/x.proto \# writes ./github.com/golang/protobuf/p/x.pb.go
```

Comment:
The display of this does not produce the code block style like the other ones.  That's why I requested for the explicit use of ```.  See https://github.com/cybrcodr/protobuf/blob/master/README.md.